examples/streaming/streaming.py

```diff
@@ -0,0 +1,60 @@
+from collections import Counter, defaultdict
+import heapq
+import wikipedia
+import ray
+
+ray.init()
```

Comment:
Would be good to move this line and the other computation into `if __name__ == "__main__"`